src/main/java/duke/command/CommandException.java

```diff
@@ -0,0 +1,7 @@
+package duke.command;
+
+public class CommandException extends Exception {
+    public CommandException(String errorMsg) {
```

Comment:
Perhaps you can write out "Message" in full? I feel that "Msg" may not be universally understood.